modules/launchd/launchd.nix

```diff
@@ -0,0 +1,869 @@
+# launchd option type from nix-darwin
+#
+# Original Source:
+# https://github.com/LnL7/nix-darwin/blob/a34dea2/modules/launchd/launchd.nix
+
+# Copyright 2017 Daiderd Jordan
+#
+# Permission is hereby granted, free of charge, to any person obtaining
```

Comment:
This file contains the definition for the Launchd option type, the code responsible for translating Nix expressions into LaunchAgent configurations.  Since this file comes from the nix-darwin project, I have included the license (MIT) and copyright headers here.

Other files aren't based on outside projects.